### PR TITLE
tests: add ms cluster case

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: ref #8683

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: tests/integrations/realcluster/cluster.go [hunk 1]
```diff
@@ -165,3 +182,58 @@ func waitTiupReady(t *testing.T, tag string) {
 	}
 	require.Failf(t, "TiUP is not ready", "tag: %s", tag)
 }
+
+type msClusterSuite struct {
```
Comment (okJiang) [inline tests/integrations/realcluster/cluster.go:186]: seems we can add a field `ms bool` to `clusterSuite` rather than create a new structure.

**Review by okJiang — COMMENTED** [review]
rest lgtm

File: tests/integrations/realcluster/cluster.go (hunk 1 above)
Comment (rleungx) [reply tests/integrations/realcluster/cluster.go]: good point

**Review by okJiang — APPROVED** [review]

**Pull request merged**